test/lib/ufe/testShaderNodeDef.py

```diff
@@ -242,8 +242,11 @@ def testNodeCreation(self):
         self.assertIsNotNone(connections)
         self.assertEqual(len(connections.allConnections()), 0)
 
-    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 6, 'NativeType metadata is only available in UFE v6 or greater')
     def testNativeTypeMetadata(self):
+        # Make sure the feature exists before attempting to test.
+        if not os.environ.get('UFE_HAS_NATIVE_TYPE_METADATA'):
+            return
+        
```

Comment:
This is okay, but it would be better as:

`@unittest.skipIf(os.getenv('UFE_HAS_NATIVE_TYPE_METADATA', 'NOT-FOUND') not in ('1', "TRUE"), 'Test only available if UFE AttributeDef 'NativeType' metadata exists.')`